### PR TITLE
Update README to reflect content build only (#16714)

### PR DESCRIPTION
**Pull request by alexandec**
## Description
Update the README to reflect that this repo only handles the content build, not the app build.

## Testing done
Review README

## Acceptance criteria
- [x] README is updated to remove irrelevant information and include useful information for content build

### CODE REVIEW

**Comment by alexandec** [comment]
@timwright12 main questions here (when you get a chance): did I remove the correct items, and is there anything that would be useful to add?

File: README.md [hunk 1]
```diff
@@ -178,13 +75,9 @@ for doing very specific things.
...
-| run the site for local development with automatic rebuilding of code and styles for specific **apps**       | `yarn watch --env.entry disability-benefits,static-pages`. Valid application names are in each app's `manifest.json` under `entryName`                                                                                       |
-| run the site for local development with automatic rebuilding of code and styles for static **content**      | `yarn watch:static`                                                                                                                                                                                                          |
+| run the site for local development with automatic rebuilding of sass **with** css sourcemaps | `yarn watch:css-sourcemaps` then visit `http://localhost:3001/`. You may also set `--env.buildtype` and `NODE_ENV` though setting `NODE_ENV` to production will make incremental builds slow.                                |
```
Comment (timwright12) [inline README.md:78]: This won't work, there's no sass in content-build. To do this kind of thing you need to run a build/watch in vets-website

File: README.md [hunk 2]
```diff
@@ -178,13 +75,9 @@ for doing very specific things.
...
 | run all linters                                                                                             | `yarn lint`                                                                                                                                                                                                                  |
 | run only javascript linter                                                                                  | `yarn lint:js`                                                                                                                                                                                                               |
 | run only sass linter                                                                                        | `yarn lint:sass`                                                                                                                                                                                                             |
```
Comment (timwright12) [inline README.md:83]: This might work, but there's no sass

**Comment by timwright12** [comment]
@alexandec Currently the site runs on port 3002 so it can run alongside vets-website on port 3001. For now I think that would be good to note even if we change it

File: README.md (hunk 1 above)
Comment (alexandec) [reply README.md:78]: Okay cool, I'll remove this then, and I'll remove the sass linter line as well (I'm assuming the JS linter could still be useful, at least occasionally)

File: README.md (hunk 2 above)
Comment (alexandec) [reply README.md:83]: Interesting, I'll put them back. Maybe they could still be useful

File: README.md (hunk 1 above)
Comment (timwright12) [reply README.md:78]: yeah there's still some JS in there, just nothing for the site itself

**Review by timwright12 — APPROVED** [review]
I think this is good for now

**Pull request merged**